include/drivers/usb/usb_dc.h

```diff
@@ -23,7 +23,7 @@
  * USB endpoint direction and number.
  */
 #define USB_EP_DIR_MASK		0x80
-#define USB_EP_DIR_IN		0x80
+#define USB_EP_DIR_IN		0x80U
 #define USB_EP_DIR_OUT		0x00
 
```

Comment:
Why only the DIR_IN and not the other defines?

Reply:
Addressed